schema/constants.go

```diff
@@ -95,16 +95,17 @@ const (
 )
 
 const (
-	CoinAPIOpenBazaarBTC = "https://btc.blockbook.api.openbazaar.org/api"
-	CoinAPIOpenBazaarBCH = "https://bch.blockbook.api.openbazaar.org/api"
-	CoinAPIOpenBazaarLTC = "https://ltc.blockbook.api.openbazaar.org/api"
-	CoinAPIOpenBazaarZEC = "https://zec.blockbook.api.openbazaar.org/api"
-	CoinAPIOpenBazaarETH = "https://rinkeby.infura.io"
+	CoinAPIOpenBazaarBTC = "https://btc.api.openbazaar.org/api"
```

Comment:
Are these the correct URLs? These are the one's that give a certificate error iirc.